duckduckgo/Android
# Update asset used when favicon unavailable to higher quality asset

Task/Issue URL: https://app.asana.com/0/193817002363848/709872649224577
Tech Design URL: 
CC: 

**Description**:
When we fail to grab a favicon for a "add to home" shortcut, we need to use a placeholder. Previously, the asset used was low quality.

**Steps to test this PR**:
1. Visit page which has no favicon (e.g., https://example.com)
1. Add to home from the overflow
1. Verify the Dax icon looks good


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
